themes/website/static/js/osdetect.js

```diff
@@ -8,9 +8,9 @@ var OSName = "Unknown";
 var defClient = {
 	title: "Wiki Downloads",
 	name: "",
-	icon: "external-link",
+	faicon: "external-link",
 	desc: false,
-	dlLink: "http://wiki.tox.chat/binaries",
+	dlLink: "http://wiki.tox.chat/binaries"
```

Comment:
A nitpick comment, but you're not very consistent about where you have a comma after the last member in the struct and where you don't.

Reply:
I guess I only noticed that one for some reason. I'm working on a PR to remove all that hardcoded client information from JS, so it won't matter anymore.